app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    commands = command.strip().split()
+
+    if commands[0] != "mv" or len(commands) != 3:
+        raise(
+            "Command must have 'mv' and exactly "
+            "2 arguments (source and destination)"
+        )
```

Comment:
The `raise` statement is missing the `Exception` type. It should be `raise ValueError(...)` or another appropriate exception type to indicate the error.